### PR TITLE
Add Penpot icon

### LINKED ISSUE #6664: Request: Penpot
### Brand Name

Penpot

### Website

https://penpot.app/

### Popularity Metric

The Alexa Rank is 297,05. See https://www.alexa.com/siteinfo/penpot.app

### Official Resources for Icon and Color

I think that the proper logo for this website can be found here: https://github.com/penpot/penpot/blob/develop/frontend/resources/images/favicon.png

Color: `#000000`. I think.

### Additional Comments

_No response_

### Contributing

No

### PR DESCRIPTION
**Pull request by adsingh14**
penpot

**Issue:** closes #6664

**Similarweb rank:**
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

Extracted from website header and edited in Inkscape, then rendered from SVGO with `precision: 3`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -8198,6 +8198,11 @@
             "hex": "CD1414",
             "source": "https://www.penny.de/"
         },
+        {
+            "title": "penpot",
+            "hex": "000000",
+            "source": "https://penpot.app/"
```
Comment (sisp) [inline _data/simple-icons.json:8204]: According to the discussion in #6664, I believe you've used the SVG file on GitHub as the source for converting the icon. If that's the case, I think that URL should be used instead.

```suggestion
            "source": "https://github.com/penpot/penpot/blob/cbe51fcabd8146391df007254b2a854bae857fe6/frontend/resources/images/icons/penpot-logo-icon.svg"
```

**Review by sisp — COMMENTED** [review]
Thanks for your contribution, @adsingh14! :pray:

Any reason why you're using a precision of 4 instead of the default 3? I've also left an inline comment regarding the source URL.

File: _data/simple-icons.json (hunk 1 above)
Comment (adsingh14) [reply _data/simple-icons.json:8204]: I extracted the logo from the official website instead of github by using svg grubber as mentioned in PR.

Comment (sisp) [reply _data/simple-icons.json:8204]: Ah, sorry for my oversight. You're right, it's mentioned in the PR. :+1:

**Comment by adsingh14** [comment]
> Thanks for your contribution, @adsingh14! :pray:
> 
> Any reason why you're using a precision of 4 instead of the default 3? I've also left an inline comment regarding the source URL.

Is there any difference between the original and this icon? If so, then I'll proceed with another icon.

**Comment by sisp** [comment]
I haven't checked yet because the default precision is 3 and I was wondering whether you've noticed a need to increase it to 4. If we can get an accurate icon with a precision of 3, that would be better.

**Comment by adsingh14** [comment]
I hope the current icon is perfectly fit for this icon.

image

File: _data/simple-icons.json (hunk 1 above)
Comment (adsingh14) [reply _data/simple-icons.json:8219]: Please have a look at https://github.com/simple-icons/simple-icons/pull/7911#issuecomment-1268152756

**Comment by sisp** [comment]
The icon might be a perfect fit, I was just wondering whether a precision of 3 would suffice without sacrificing accuracy. I.e.: If you optimized the icon to a precision of 3, would the fit be good, too?

File: _data/simple-icons.json [hunk 2]
```diff
@@ -8219,6 +8219,11 @@
             "hex": "CD1414",
             "source": "https://www.penny.de/"
         },
+        {
+            "title": "penpot",
```
Comment (LitoMore) [inline _data/simple-icons.json:8223]:
```suggestion
            "title": "Penpot",
```

Typo.

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 2 above)
Comment (adsingh14) [reply _data/simple-icons.json:8223]: Here, I used the logo text as the title.

Comment (LitoMore) [reply _data/simple-icons.json:8223]: See here to get to know how they spell it: https://penpot.app/about.html

Comment (LitoMore) [reply _data/simple-icons.json:8223]: > They should keep the same case for the logo.

No. There are different things. They can spell it to anything in the logo they want.

**Review by LitoMore — APPROVED** [review]
CleanShot 2022-10-06 at 18 08 01@2x

@adsingh14 Looks good. Thanks!

**Review by LitoMore — APPROVED** [review]
CleanShot 2022-10-06 at 18 35 41@2x

@adsingh14 Yes. Precison 3 looks good as well.

**Pull request merged**